src/components/Card/styled.ts

```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components';
+import { Button } from 'components/Button';
+
+export const CardContainer = styled.div`
+  height: 720px;
+  width: 1080px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: ${({ theme }) => theme.colors.alternativeText};
+  box-shadow: 10px 0px 10px -5px ${({ theme }) => theme.colors.shadow};
+  background: ${({ theme }) => theme.colors.primaryText};
```

Comment:
https://images.app.goo.gl/6PFsyLV5QvViJfKQ7 Postarajcie się zrobić coś takiego.

border-radius na card
Serio te struktury komponentów to mega dziwna logika, Czemu wszystkie detale i oceny są jako <li> , Czemu korzystaliście z klasowych komponentów? Mam wrażenie że albo korzystaliście z jakichś mega starych przykładów albo próbowaliście sami to zrobić, tylko że nie macie na tyle dużej znajomości technologii żeby to zrobić, wtedy serio najlepiej zerknąć na jakiś przykład jak to wygląda na necie i w aplkacjach.